packages/next-app/src/app/metadata/layout.tsx

```diff
@@ -19,7 +22,27 @@ export default function Layout({ children }: { children: React.ReactNode }) {
         linkUrl={"/metadata"}
         title={"Pool Metadata"}
         imageSrc={balancerSymbol}
-      />
+      >
+        <Dialog title={"Search pool"} content={<SearchPoolForm />}>
+          <div className="pointer-events-auto relative rounded bg-white">
```

Comment:
Curious about why you chose to use white here. Did you want to call attention to the lighter element on a dark background?

Reply:
I believe the white background would catch the user's attention, but I'm not confident with my design skills 😆

Reply:
I think it should be a different shade of the background color not to call too much attention to it